downloader/src/main/kotlin/vcs/GitWorkingTree.kt

```diff
@@ -95,7 +95,7 @@ open class GitWorkingTree(workingDir: File, private val gitBase: GitBase) : Work
             val remoteForCurrentBranch = BranchConfig(repo.config, repo.branch).remote
 
             val remote = if (remotes.size <= 1 || remoteForCurrentBranch == null) {
-                remotes.firstOrNull()
+                remotes.find { it.name == "origin" } ?: remotes.firstOrNull()
```

Comment:
Just wondering, does this actually solve any real-world problem? Because the code should only be relevant if `remoteForCurrentBranch == null`, and I'm wondering when it's the case that the remote for the current branch could not be determined, *and* we have multiple remotes.

Reply:
I noticed that will working with the analyzer tests it was always using the URL of an internal fork instead of the GitHub repository for the project VCS. This is not a problem because we always patch our expected results, but it is inconvenient.
Also, when running the analyzer on a local repository with multiple remotes, I think in general it makes more sense to use origin for the project's VCS info instead of a random remote.

Reply:
BTW, this somehow rang a bell on my side, and I just found that [we had it like that before](https://github.com/oss-review-toolkit/ort/pull/2087/files#diff-6a0e0bb8886076885391205cc0c9bef5L86) my port to JGit. So thanks for fixing it up.